distribution/changelog.txt

```diff
@@ -1,4 +1,4 @@
-0.1.2 (in development)
+﻿0.1.2 (in development)
```

Comment:
Looks like a BOM was added to the beginning of the file. Is that alright?

Reply:
I don't think so. This file gets read by game when the version has changed, I imagine it can cause issues.